src/main/kotlin/io/ipfs/kotlin/commands/Add.kt

```diff
@@ -40,9 +60,17 @@ class Add(val ipfs: IPFSConnection) {
                 .build();
 
         val response = ipfs.okHttpClient.newCall(request).execute().body()
-        val result = adapter.fromJson(response.source())
+        var result: NamedHash = NamedHash("nameprobe","hashprobe")
+
+//        val result = adapter.fromJson(response.source())
```

Comment:
(which is to say: I couldn't make the rebase work out-of-order, so these debugging lines make no sense. Sorry!)